File: 백준/Bronze/2576. 홀수/홀수.java
import java.util.ArrayList;
import java.util.Scanner;
import java.util.Collections;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        ArrayList<Integer> res = new ArrayList<>();

        for (int i = 0; i < 7; i++) {
            int n = sc.nextInt();
            if (n % 2 != 0)
                res.add(n);
        }
        sc.close();

        if (res.isEmpty()) {
            System.out.println(-1);
        } else {
            int sum = 0;
            for (int num : res) {
                sum += num;
            }
            System.out.println(sum);
            System.out.println(Collections.min(res));
        }
    }
}

File: 백준/Bronze/2576. 홀수/홀수.cs
List<int> res = new List<int>();

for (int i = 0; i < 7; i++)
{
    int n = int.Parse(Console.ReadLine());
    if (n % 2 != 0) 
        res.Add(n);
}

if (res.Count == 0)
    Console.WriteLine(-1);
else
{
    Console.WriteLine(res.Sum());
    Console.WriteLine(res.Min());
}